src/generators/json/constants.mjs

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+// Grabs the default value if present
+export const DEFAULT_EXPRESSION = /^(?:D|d)efault(?:s|):$/;
+
+// Grabs the type and description of one of the formats for event types
+export const EVENT_TYPE_DESCRIPTION_EXTRACTOR = /{(.*)}(.*)/;
+
+// Grabs type and optional description for a method's parameter signature
+export const METHOD_TYPE_EXTRACTOR = /^{(.*)}( .*)?$/;
+
+// Grabs return type and optional description for a method signature
+// Accepts the following:
+// Returns: {string}
+// Returns {string}
+// Returns: {string} bla bla bla
+export const METHOD_RETURN_TYPE_EXTRACTOR = /^(?:R|r)eturns(:?) {(.*)}( .*)?$/;
```

Comment:
I'd rather do a case insensitive search

Reply:
This isn't just checking if it starts with `Returns:` though, it's extracting the type information and possible description from it as well

Reply:
Can we change it to `/^Returns(:?) {(.*)}( .*)?$/i`?